server/models/Expense.js

```diff
@@ -480,15 +481,26 @@ function defineModel() {
     return legacyPayoutMethod === 'paypal' ? PayoutMethodTypes.PAYPAL : PayoutMethodTypes.OTHER;
   };
 
-  Expense.getMostPopularExpenseTagsForCollective = async function (collectiveId, limit = 100) {
+  Expense.getCollectiveExpensesTags = async function (collectiveId, limit = 100) {
+    const noTag = 'no tag';
     return sequelize.query(
       `
-      SELECT UNNEST(tags) AS id, UNNEST(tags) AS tag, COUNT(id)
-      FROM "Expenses"
-      WHERE "CollectiveId" = $collectiveId
-      AND "deletedAt" IS NULL
-      GROUP BY UNNEST(tags)
-      ORDER BY count DESC
+      SELECT TRIM(UNNEST(COALESCE(e."tags", '{"${noTag}"}'))) AS id,
+      TRIM(UNNEST(COALESCE(e."tags", '{"${noTag}"}'))) AS tag,
+      COUNT(e."id") as "count",
+      ABS(SUM(t."amountInHostCurrency")) as "amount",
```

Comment:
We should convert this to collective currency.

Reply:
Now looking at collective `currency`